atomic_reactor/schemas/container.json

```diff
@@ -4,229 +4,376 @@
   "title": "container image build configuration",
 
   "type": ["object", "null"],
-  "properties": {
-    "go": {
-      "type": "object",
-      "properties": {
-        "modules": {
-          "type": ["array", "null"],
-          "items": {
-            "type": "object",
-            "properties": {
-              "module": {
-                "type": "string",
-                "description": "Top-level Go module (package) name which will be built",
-                "examples": [
-                  "example.com/go/packagename",
-                  "example.com/go/anotherpackage"
-                ]
+  "anyOf": [
```

Comment:
Seems fine, I was thinking oneOf but it makes little difference in this instance.

Reply:
I can't use oneOf, because IF you have for example only Platforms with oneOf it isn't able to decide which schema variant to use, so we need anyOf